### ISSUE TITLE
1.15.72 does not render window contents when switching workspace in XMonad

### ISSUE THREAD

**Issue by srhoulam** [brave/brave-browser#12058]
## Description 

Under a tiling window manager (XMonad), switching workspace to Brave results in Brave not rendering window contents, and just appearing to be a still image of the workspace I just left. Layout doesn't matter, this will occur with all visible Brave windows. I include a screenshot of two Brave windows and one terminal emulator (gnome-terminal) demonstrating this issue. While the Brave windows are not rendering their contents, they are still responsive to interaction, e.g., clicks and keystrokes.

## Steps to Reproduce

   1. Use XMonad as your window manager. Possibly any tiling window manager.
   2. Run Brave.
   3. Switch workspaces away from Brave, and back to Brave.

## Actual result:

https://imgur.com/a/cD8RPCv

## Expected result:
The Brave window(s) will show a still image of what was on-screen before changing workspace to them.

## Reproduces how often: 

Every single time.

## Brave version (brave://version info)

Brave | 1.15.72 Chromium: 86.0.4240.75 (Official Build) (64-bit)
-- | --
Revision | c69c33933bfc72a159aceb4aeca939eb0087416c-refs/branch-heads/4240@{#1149}
OS | Linux

## Version/Channel Information:

- Can you reproduce this issue with the current release?

Never until current version.

- Can you reproduce this issue with the beta channel?

Haven't tried.

- Can you reproduce this issue with the nightly channel?

Haven't tried.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?

No.

- Does the issue resolve itself when disabling Brave Rewards?

No.

- Is the issue reproducible on the latest version of Chrome? 

No. Tested with Chromium Version 85.0.4183.121 (Official Build).

## Miscellaneous Information:

I at first suspected it could be an issue with the compositor. I disabled my compositor (compton) and the issue persisted.

**Comment by bsclifton** [comment]
I suspect this may be related to GPU/driver - can you please try disabling Hardware acceleration? You can find/toggle it under brave://settings/system

**Comment by srhoulam** [comment]
@bsclifton Yes, disabling hardware acceleration and relaunching completely put a stop to the issue.

**Comment by trenc** [comment]
Seems related to this issue:
New windows (not tabs), PWAs and shortcuts opened besides a window do not render. By disabling the harware acceleration this issue is gone too.

**Comment by bsclifton** [comment]
Hi folks - I am going to close this issue as it's a bit old without an update

I did put together some notes and a [new process for when folks are having GPU issues](https://github.com/brave/brave-browser/wiki/GPU-issues---incident-reports).

If you're reading this and you are having GPU issues, you can help me by creating a new issue here describing the problem on GitHub and then attaching the report from brave://gpu.
